portal/assets/css/register.css

```diff
@@ -0,0 +1,81 @@
+@CHARSET "UTF-8";
```

Comment:
Forcing a charset always makes me nervous. You just never know what folks will do. Is this needed?